clean up docs todos (#6886)
RasaHQ/rasa: docs/docs/messaging-and-voice-channels.mdx

* [Custom Connectors](./connectors/custom-connectors.mdx)

## Testing Channels on Your Local Machine

If you're running Rasa X Local mode or a Rasa Open Source server on `localhost`, 
most external channels won't be able to find your server URL, since `localhost` is not open to the internet. 

To make a port on your local machine publicly available on the internet, 
you can use [ngrok](https://ngrok.com/).

After installing ngrok, run:

```bash
ngrok http 5005; rasa run
```

> When you follow the instructions to make your assistant available on a channel, use the ngrok URL:

Reviewer: ```suggestion
When you follow the instructions to make your assistant available on a channel, use the ngrok URL.
```

> wherever the instructions say to use `https://<host>:<port>/webhooks/<CHANNEL>/webhook`,

Reviewer: ```suggestion
Wherever the instructions say to use `https://<host>:<port>/webhooks/<CHANNEL>/webhook`,
```

> use `https://yyyyyy.ngrok.io/webhooks/<CHANNEL>/webhook`, replacing `yyyyyy` with the randomly generated

Reviewer: ```suggestion
use `https://<url>.ngrok.io/webhooks/<CHANNEL>/webhook`, replacing `<url>` with the randomly generated
```

Author: Mm that chunk itself is not a url though 🤔

> Wherever the instructions say to use `https://<host>:<port>/webhooks/<CHANNEL>/webhook`,
> use `https://yyyyyy.ngrok.io/webhooks/<CHANNEL>/webhook`, replacing `yyyyyy` with the randomly generated
> sequence displayed in the ngrok URL in your terminal.

Author: ```suggestion
Specifically, wherever the instructions say to use `https://<host>:<port>/webhooks/<CHANNEL>/webhook`,
use `<ngrok_url>/<CHANNEL>/webhook`, replacing `<ngrok_url>` with the randomly generated
URL displayed in your ngrok terminal window. For example, if connecting your bot to Slack, 
your URL should resemble `https://26e7e7744191.ngrok.io/webhooks/slack/webhook`.
```

Author: alternative to https://github.com/RasaHQ/rasa/pull/6886#discussion_r499749534
